wnfs/src/private/node/node.rs

```diff
@@ -610,24 +574,45 @@ mod tests {
         let forest = &mut Rc::new(PrivateForest::new());
         let store = &mut MemoryBlockStore::new();
 
-        let file = PrivateFile::with_content(
+        let file = Rc::new(
+            PrivateFile::with_content(
+                Namefilter::default(),
+                Utc::now(),
+                content.to_vec(),
+                forest,
+                store,
+                rng,
+            )
+            .await
+            .unwrap(),
+        );
+
+        let mut directory = Rc::new(PrivateDirectory::new(
             Namefilter::default(),
             Utc::now(),
-            content.to_vec(),
-            forest,
-            store,
             rng,
-        )
-        .await
-        .unwrap();
+        ));
+
+        directory
+            .mkdir(&["music".into()], true, Utc::now(), forest, store, rng)
+            .await
+            .unwrap();
 
-        let file = PrivateNode::File(Rc::new(file));
-        let private_ref = file.store(forest, store, rng).await.unwrap();
+        let file_node = PrivateNode::File(file.clone());
+        let dir_node = PrivateNode::Dir(directory.clone());
```

Comment:
Is there a difference between `file.clone()` and `Rc::clone(file)`? And did you intend to use `file.clone()` here?

In any case - these are tests, so doesn't really matter.

Reply:
No diff. I should probably use `Rc::clone(file)`